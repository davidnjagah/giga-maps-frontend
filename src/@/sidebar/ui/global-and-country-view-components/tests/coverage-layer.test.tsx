import { render, screen } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import CoverageLayer from '../coverage-layer/coverage-layer';
describe('CoverageLayer', () => {

  it('should render component', () => {
    const { asFragment } = render(testWrapper(<CoverageLayer />));
    expect(asFragment).toMatchSnapshot();
  });

  // it('should render title', () => {
  //   render(testWrapper(<CoverageLayer />));

  //   const text = screen.getAllByText('schools mapped', { trim: true, exact: false });
  //   expect(text[0]).toBeInTheDocument();
  // });

});
