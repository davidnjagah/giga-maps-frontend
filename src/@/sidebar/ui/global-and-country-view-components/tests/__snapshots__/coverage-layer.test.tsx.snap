// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoverageLayer should render component 1`] = `[Function]`;
